lib/model/yake_keywords.py

```diff
@@ -26,15 +44,27 @@ def run_yake(self, text: str,
         :param num_of_keywords: int
         :returns: str
         """
+        ### if language is set to "auto", auto-detect it.
+        if language == 'auto':
+            language = detect(text)
+        ### replace special characters
+        text.replace("`", "'")
+        text.replace("‘", "'")
+        text.replace("“", "\"")
```

Comment:
These lines have no effect. `str.replace` does not modify the original string. It returns a new string; so, these lines need to be `text = text.replace(...)` . Please write test code and verify it works.

We need to replace `“` and `”` . Consider having a `replacements` dictionary that can be looped over. e.g.,
```
replacement = { "‘": "'", ....}
for k,v in replacement.items():
    text=text.replace(k,v)
```